### ISSUE TITLE
Add check to CI that MSVS files are valid

### ISSUE THREAD

**Issue by langevin-usgs** [MODFLOW-USGS/modflow6#953]
The MSVS files for MODFLOW 6 and the utilities are not checked as part of the CI testing.  Consequently, we may break them without knowing as we add new functionality.

For the Windows CI testing, we should add a workflow that builds mf6 and the utilities using the MSVS project files.

**Comment by Hofer-Julian** [comment]
Sounds good.

**Comment by Hofer-Julian** [comment]
I tried to add one, but I wasn't successful.
Here are a couple of hints for the next one giving it a try.

At the moment, our msvs files can only be build with Visual Studio 2019.
Neither Visual Studio 2020 nor msbuild can be used.

One way to get that on your CI is to use the `windows-2019`  image (instead of `windows-latest`) and then setting up a job like that:

```yaml
      - name: Setup MSVC
        uses: bus1/cabuild/action/msdevshell@v1
        with:
          architecture: x64

      - name: Build modflow6
        run: |
          devenv.exe msvs/mf6.sln /Build Release
```

In theory that should work but the last command runs until it times out.

**Comment by wpbonelli** [comment]
Invoking `devenv.exe` in CI does not hang in my testing, but it [fails on attempting to upgrade/migrate the project](https://github.com/modflowpy/install-intelfortran-action/actions/runs/6266166542/job/17016541615#step:6:294). I saw this same error locally until I followed [these steps](https://community.intel.com/t5/Intel-Fortran-Compiler/How-to-migrate-vjproj-file-from-Visual-2017-or-2019-so-that-it/m-p/1385749) to wipe all traces of VS from the system, reinstall it, and then reinstall oneAPI. After that, building with `devenv.exe` works locally (VS2019 16.11.30, oneAPI 2022.3.1).

My next step is to try the cleanup/reinstall approach in CI.
